cmd/dashboard.go

```diff
@@ -0,0 +1,104 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+
+	"github.com/dapr/cli/pkg/kubernetes"
+	"github.com/dapr/cli/pkg/print"
+	"github.com/pkg/browser"
+	"github.com/spf13/cobra"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const (
+	// dashboardSvc is the name of the dashboard service running in cluster
+	dashboardSvc = "dapr-dashboard"
+
+	// defaultHost is the default host used for port forwarding for `dapr dashboard`
+	defaultHost = "localhost"
+
+	// defaultPort is the default port used for port forwarding for `dapr dashboard`
+	defaultPort = 8080
+)
+
+var open bool
+
+var DashboardCmd = &cobra.Command{
+	Use:   "dashboard",
+	Short: "Runs the Dapr dashboard on a Kubernetes cluster",
+	Run: func(cmd *cobra.Command, args []string) {
+		config, err := kubernetes.GetKubeConfig()
+
+		if err != nil {
+			print.FailureStatusEvent(os.Stdout, "Failed to initialize kubernetes client")
+		}
+
+		print.InfoStatusEvent(os.Stdout, "Launching Dapr dashboard in kubernetes cluster")
+
+		err = kubernetes.InitDashboard()
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Failed to initialize dashboard")
+			return
+		}
+
+		signals := make(chan os.Signal, 1)
+		signal.Notify(signals, os.Interrupt)
+		defer signal.Stop(signals)
+
+		portForward, err := kubernetes.NewPortForward(
+			config,
+			meta_v1.NamespaceDefault,
+			dashboardSvc,
+			defaultHost,
+			defaultPort,
+			defaultPort,
+			false,
+		)
+		if err != nil {
+			print.FailureStatusEvent(os.Stderr, "Failed to initialize port forwarding: %s\n", err)
```

Comment:
It would be better if either you can change the error message here to "creating.." or change Line 70 to "error in port forwarding.."
because both messages conveying the same message for 2 different related things..